cdap-ui/app/cdap/components/DataPrep/index.js

```diff
@@ -64,7 +69,7 @@ export default class DataPrep extends Component {
       DataPrepStore.dispatch({
         type: DataPrepActions.reset
       });
-      let workspaceId = this.props.singleWorkspaceMode ? this.props.workspaceId : cookie.load('DATAPREP_WORKSPACE');
+      let workspaceId = this.props.workspaceId ? this.props.workspaceId : cookie.load('DATAPREP_WORKSPACE');
```

Comment:
Might be a prpblem if users go to `/dataprep` in a browser with no cookie set. Right now I am seeing 3 tabs none selected and the tab content is `Please create a workspace to wrangle data`. Its still ok but since we are moving away from users creating workspace might be good to avoid it.